### ISSUE TITLE
Resource count and size are not displayed correctly

### ISSUE THREAD

**Issue by mzur** [ArunaStorage/ArunaServer#141]
I created a project and a collection. Then I've uploaded a directory to the collection (via S3), which was created as a "dataset" with several "object" children. In the web UI, the project, collection and dataset always show "count 1" and "size 0 B". Only a single object shows an actual size.

**Comment by St4NNi** [comment]
Thanks for reporting. I have moved this to the Server repo because technically this is not a visual bug. 
Re-calculation of sizes / counts is **not implemented** yet for higher level resources. 

There were on-going discussions if it is feasible to update this ad-hoc, the current proposed solution will implement a cron that updates stats periodically (~every 5 minutes), so that they will eventually show the correct stats. Immediate updates would have a cascading effect for higher level resources that might drastically hurt performance especially if many objects get uploaded in parallel.

**Comment by das-Abroxas** [comment]
He Martin, 
we have now implemented a solution that corresponds to the proposal already presented by @St4NNi. 

The statistics of higher level resources are now updated at a fixed interval (currently 15 seconds) to avoid the possible cascading effect of on-demand updates. In addition, the ArunaServer instances synchronize with each other when the last update was started to avoid nonsensical concurrent attempts to start the update. 

I'm closing this issue for now, but feel free to leave any kind of feedback here. We'd appreciate it.
